### PR TITLE
Upgrade PyYAML

### PR DESCRIPTION
**Pull request by piyush-garg**
This will remove the installed PyYAML and
installs the latest

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: openshift/Dockerfile [hunk 1]
```diff
@@ -14,5 +14,11 @@ RUN OPENSHIFT_CLIENT_VERSION=$(curl -s https://mirror.openshift.com/pub/openshif
     && rm -rf ./openshift* \
     && oc version
 
+# remove PyYAML 3.10, install pip and install latest PyYAML
+RUN yum remove -y PyYAML \
+    && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py \
+    && python get-pip.py \
+    && pip install PyYAML
```
Comment (chmouel) [inline openshift/Dockerfile:21]: I believe you can simply do :
 
```suggestion
RUN python3 -mpip install -U PyYAML
```

image

**Review by chmouel — CHANGES_REQUESTED** [review]
Let's use python directly to update pyyaml

File: openshift/Dockerfile (hunk 1 above)
Comment (piyush-garg) [reply openshift/Dockerfile:21]: Nice, thanks for the suggestion.

**Comment by chmouel** [comment]
I am confused, is the change was tested?

**Pull request closed without merging**